existing-communities/README.md

```diff
@@ -1 +1,47 @@
+## Existing Communities
+Before we can compare different communities and their structure, we first need to define a few (types of) communities. We will look at two types of real-life communities and two types of virtual communities, such that we can discuss their similarities and differences.
+### Ecosystems
+An ecosystem can be seen as a natural form of a community. It is bound to a location where a diversity of animals and species interact with eachother{{"EcosysDef" | cite}}. Although there is no real communication, the combination of individual self-interest (the survival of their offspring or themself) creates a network of dependencies and interactions that one can interpret as a community.  
+A key factor of an ecosystem is stability. In a stable ecosystem{{"EcosysSDef" | cite}}, each species acts in such a way that they directly enable or inderectly do not interfere with the survival of another species. . An ecosystem will remain stable as long as the system is able to deal with outside disturbances.
+
+### Intentional Communities
+Intentional communities are man-made communities, where its members attempt to realise a common alternative way of life outside of mainstream society{{"meijering2007intentional" | cite}}. Although there is no one list of criteria to characterise them, the most cited criteria are:
+
+1. No bonds by familial relationships only.
+2. A minimum of three to five adult members.
+3. Members join voluntarily
+4. Geographical and psychological separation from mainstream society.
+5. A common ideology that is adhered to by all members.
+6. Sharing of (a part of) one's property.
+7. The interest of the group prevails over individual interests.
+
+Not all intentional communities fulfill all of these criteria but they give a good idea of what is and what is not a intentional community. Intentional communities can be quite diverse. To get a better understanding of the different ways an intentional community can work, they can be split into four groups: ecological, communal, religious and practical communities{{"meijering2007intentional" | cite}}.
+
+**Ecological Communities**  
+Ecological communities are communities that withdraw to remote locations in an effort to live according to their own ideals. They actively separate themself from society and aim to be self-sufficient. The separation is what identifies this type of intentional communities{{"meijering2007constructing" | cite}}{{"meijering2007intentional" | cite}}.  
```

Comment:
> **The members** actively **seperate themselves**

Or, like before, make it singular again: `An ecological community is a community that withdraws to a remote location in an effort to live according to their own ideals. The members actively **seperate themselves**